content/en/docs/concepts/configuration/secret.md

```diff
@@ -290,7 +290,7 @@ ConfigMap.
 
 ### TLS secrets
 
-Kubernetes provides a builtin Secret type `kubernetes.io/tls` for to storing
+Kubernetes provides a builtin Secret type `kubernetes.io/tls` for storing
```

Comment:
Great catch!!!

Reply:
Thanks Brad! It's small, but I'm glad to have finally contributed to OSS 😄